### DISCUSSION TITLE
Error 1208 - Failed to create an object

### DISCUSSION THREAD

**Discussion by SchmidtStefan** [HCL-TECH-SOFTWARE/domino-online-meeting-integration#107] [Q&A]
We have one user receiving the above error messag when creating a Teams Meeting. It is only one user getting this error, it works for everybody else.

Does someone here know the reason for this error and how to fix it?

Thank you

/Stefan

**Comment by yvderlet** [comment]
Hello,

I have the same issue...

Did you get an answer ?

**Reply by SchmidtStefan** [reply]
No, I didn't get an answer.
